mne/preprocessing/ica.py

```diff
@@ -254,7 +254,8 @@ def __repr__(self):
               hasattr(self, 'n_components_') else
               'no dimension reduction')
         if self.info is not None:
-            ch_fit = ['"%s"' % c for c in ['mag', 'grad', 'eeg'] if c in self]
+            ch_fit = ['"%s"' % c for c in ['mag', 'grad', 'eeg', 'seeg']
```

Comment:
although maybe we'd need a second one as `_DATA_TYPES = ('meg', 'eeg', 'seeg')` and call the other one `_DATA_TYPES_SPLIT` or something (since MEG is split)